docs/configuration/dataValue.md

```diff
@@ -0,0 +1,40 @@
+# DataValue
+
+Prints the value of the point on the graph
+
+## Usage
+To enable printing of the data values, add `drawValue = true` to the dataset. 
+
+## Configuration
+The datavalue configuration is passed into the `options.valueLabel` namespace. The global options for this is defined in `Chart.defaults.global.valueLabel`.
```

Comment:
Might be a good idea to start moving options for *new* plugins into the `options.plugins.{id}` object as we have for the [filler plugin](/chartjs/Chart.js/blob/master/src/plugins/plugin.filler.js#L9), in this case, `Chart.defaults.global.plugins.datalabels` (@etimberg, thoughts?)

And so: 

```javascript
new Chart(ctx, {
    options: {
        plugins: {
            datalabels: {
                 //...
            }
        }
    }
}
```

See the [plugin documentation](/chartjs/Chart.js/blob/master/docs/developers/plugins.md#plugin-options)